srcpkgs/runit-void/files/09-apparmor.sh

```diff
@@ -18,15 +18,9 @@ if [ -n "$APPARMOR" ]; then
 	[ "$APPARMOR" = "complain" ] && AACOMPLAIN="-C"
 
 	if [ -d /etc/apparmor.d -a -x /usr/bin/apparmor_parser ]; then
-		for profile in /etc/apparmor.d/*; do
-		case "$profile" in
-			*.new-*_*) continue ;;
-		esac
-		if [ -f "$profile" ]; then
-			printf '* Load profile %s: %s\n' "($APPARMOR)" "$profile"
-			apparmor_parser -a $AACOMPLAIN "$profile"
-		fi
-		done
+		find /etc/apparmor.d -maxdepth 1 -type f ! -name '*.new-*_*' |\
+			xargs -d"\n" -I{} --max-procs="$(grep -c ^processor /proc/cpuinfo)" \
```

Comment:
`nproc` https://www.flamingspork.com/blog/2020/11/25/why-you-should-use-nproc-and-not-grep-proc-cpuinfo/